src/main/java/hw19/ValueCalculator.java

```diff
@@ -0,0 +1,49 @@
+package hw19;
+
+import java.util.Arrays;
+
+public class ValueCalculator {
+    private final  float[] array;
+    private static final int SIZE = 1_000_000;
+    private static final int HALF_SIZE = SIZE / 2;
+
+    public ValueCalculator() {
+        this.array = new float[SIZE];
+    }
+
+    public long calculateFillingTime(int value) {
+        long startTime = System.currentTimeMillis();
+        fillArray(value);
+        long endTime = System.currentTimeMillis();
+        return endTime - startTime;
+    }
+
+    private synchronized void fillArray(int value) {
+        Arrays.fill(array, value);
```

Comment:
It is not necessary to fill the array with an initial value as it's already filled with zeroes by default for all numeric types.